apps/swirl-docs/src/components/Documentation/Parameter.tsx

```diff
@@ -2,33 +2,54 @@ import { Tag } from "../Tags";
 import { ReactMarkdown } from "react-markdown/lib/react-markdown";
 import { Text } from "../swirl-recreations";
 import classNames from "classnames";
+import { ReactNode, useState } from "react";
+import { SwirlIconAdd, SwirlIconRemove } from "@getflip/swirl-components-react";
 
 interface ParameterProps {
+  children?: ReactNode;
   name: string;
-  type: string;
+  type?: string;
   required?: boolean;
   description?: string;
+  enumValues?: string[];
 }
 
 export function Parameter({
+  children,
   name,
   type,
   description,
   required,
+  enumValues,
 }: ParameterProps) {
+  const [expanded, setExpanded] = useState(false);
```

Comment:
**nitpick:** in most components i try to handle boolean variables with `is` or `has` prefixes.

so here isExpanded and setIsExpanded

Reply:
Done.